### PR TITLE
Created the homepage with navigation to the 3 main pages

### PR DESCRIPTION
**Pull request by theslic**
Created the home page with navigation to the main 3 pages: dashboard, monitoring, and submission each with a short description of the page. The page should direct the user to the login page when selecting one of 3 options.

### CODE REVIEW

File: src/Presentation/Components/Pages/Home.razor [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
 <PageTitle>Home</PageTitle>
-<div class="d-flex justify-content-center"><h1>DESIMONE CAPSTONE PROJECT</h1></div>
-<div class="d-flex justify-content-center"><h3>Cloud Compute Job Management System</h3></div>
-<div class="d-flex justify-content-center"><h4>Team 2</h4></div>
+<div class="d-flex justify-content-center"><h1>Cloud Compute Job Management System</h1></div><br />
+<div class="d-flex justify-content-center"><h3>Simplifying job submission with real-time</h3></div>
+<div class="d-flex justify-content-center"><h3>progression tracking and cost management</h3></div><br />
+ 
+
+<!-- CenteredBoxesWithButtonsAndDescriptions.razor -->
```
Comment (ducmvo) [inline src/Presentation/Components/Pages/Home.razor:13]: remove this?

File: src/Presentation/Components/Pages/Home.razor [hunk 2]
```diff
@@ -1,6 +1,34 @@
 ﻿@page "/"
+@inject NavigationManager NavigationManager
+@using Microsoft.AspNetCore.Authorization
+@using Presentation.Components.UI
+
```
Comment (ducmvo) [inline src/Presentation/Components/Pages/Home.razor:5]: these are not being used, please remove

File: src/Presentation/Components/Pages/Home.razor [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+<div class="centered-container">
+	<div class="box">
+		<a href="dashboard" class="home-button">
+			<h1>Dashboard</h1>
+			<p>The dashboard provides a concise summary of key metrics for convenient viewing.</p>
+		</a>
+	</div>
+	<div class="box">
+		<a href="submission" class="home-button">
```
Comment (ducmvo) [inline src/Presentation/Components/Pages/Home.razor:23]: please replace `a` tag with supported NavLink for navigation
```suggestion
		<NavLink href="dashboard" class="home-button">
```

File: src/Presentation/Components/Pages/Home.razor.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+@font-face {
+    font-family: 'Trade Gothic Light';
+    src: url('fonts/TradeGothicLight.ttf') format('ttf');
+}
+/* Add some basic styling */
+.centered-container {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    height: 100vh;
```
Comment (ducmvo) [inline src/Presentation/Components/Pages/Home.razor.css:30]: 100vh will add extra space to the bottom leads to scollable at initial rendering. Can we fix this?

File: src/Presentation/Components/Pages/Home.razor.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        top: 70%;
+        left: 0;
+        right: 0;
+        transform: translateY(-50%);
+        margin: 10px;
+        font-size: 16px;
+        color: #fff;
+        font-family: 'Trade Gothic Light';
+    }
+
```
Comment (ducmvo) [inline src/Presentation/Components/Pages/Home.razor.css:79]: Can we use bootstrap instead of custom css?

**Review by ducmvo — CHANGES_REQUESTED** [review]
We can replace custom css and use bootstrap instead.
For example:
```html
<div class="container d-flex flex-column justify-content-center align-items-center">
    <div class="d-flex justify-content-center mt-xl-5"><h1>Cloud Compute Job Management System</h1></div><br />
    <div class="d-flex justify-content-center"><h3>Simplifying job submission with real-time</h3></div>
    <div class="d-flex justify-content-center mb-xl-5"><h3>progression tracking and cost management</h3></div><br />

    <div class="container d-flex flex-row gap-lg-5 mt-xl-5">
        <div class="card">
            <NavLink class="btn btn-outline-dark py-xl-5 px-5 h-100" href="dashboard">
                <h3 class="mb-5">Dashboard</h3>
                <p>The dashboard provides a concise summary of key metrics for convenient viewing.</p>
            </NavLink>
        </div>
        <div class="card">
            <NavLink class="btn btn-outline-dark py-xl-5 px-5 h-100" href="submission">
                <h3 class="mb-5">Submission</h3>
                <p>The submission page is where users input various job details for calculation.</p>
            </NavLink>
        </div>
        <div class="card">
            <NavLink class="btn btn-outline-dark py-xl-5 px-5 h-100" href="monitoring">
                <h3 class="mb-5">Monitoring</h3>
                <p>The monitoring page is where users can view the results of various calculations.</p>
            </NavLink>
        </div>
    </div>
</div>
```

**Review by giaou — APPROVED** [review]

**Pull request closed without merging**